content/applications/marketing/marketing_automation/getting_started/testing_running.rst

```diff
@@ -1,41 +1,50 @@
-==============================
-Testing and running a campaign
-==============================
+=========================
+Testing/running campaigns
+=========================
 
 Odoo gives users the ability to test marketing campaigns (and mailings) before officially running
 them.
 
-First, open the :guilabel:`Marketing Automation` application and click on a campaign. Make sure
-the campaign already has activities configured on it (or build a campaign by following the
-directions here on :doc:`this documentation <workflow_activities>`). To start a test, click the
-:guilabel:`Launch a Test` button at the top of the template form.
+First, open the *Marketing Automation* application, and select the desired campaign, which reveals
+that campaign's detail form.
+
+On the campaign detail form, make sure the campaign already has activities configured (or build a
+campaign by following the directions here on :doc:`this documentation <workflow_activities>`).
```

Comment:
Use global relative link instead of local relative
```suggestion
campaign by following the directions here on :doc:`this documentation
</applications/marketing/marketing_automation/getting_started/workflow_activities>`).
```